docs/copied-from-beats/docs/security/users.asciidoc

```diff
@@ -197,156 +344,245 @@ information.
 |Send monitoring data to the monitoring cluster
 |====
 
-. Assign the following role to users who will view the monitoring data in
-{kib}:
+TIP: See <<monitoring-metricbeat-collection>>
+for complete details on setting up Metricbeat collection.
+--
+
+[float]
+[[privileges-to-publish-monitoring-view]]
+==== View monitoring data
+
+To grant users the required privileges for viewing monitoring data:
+
+. Create a *monitoring role*, called something like
++{beat_default_index_prefix}_monitoring_viewer+, that has the following privileges:
++
+[options="header"]
+|====
+|Type | Privilege | Purpose
+
+| Spaces
+|`Read` on Stack monitoring
+|Read-only access to the Stack Monitoring feature in {kib}.
+
+| Spaces
+|`Read` on Dashboards
+|Read-only access to the Dashboards feature in {kib}.
+|====
++
+. Assign the *monitoring role*, along with the following built-in roles, to users who
+need to view monitoring data for {beatname_uc}:
 +
 [options="header"]
 |====
 |Role | Purpose
 
 |`monitoring_user`
-|Use *Stack Monitoring* in {kib} to monitor {beatname_uc}
+|Grants access to monitoring indices for {beatname_uc}
 |====
---
-endif::serverless[]
 
-[[privileges-to-publish-events]]
-==== Grant privileges and roles needed for publishing
+////
+***********************************  ***********************************
+***********************************  ***********************************
+////
 
-Users who publish events to {es} need to create and write to {beatname_uc}
-indices. To minimize the privileges required by the writer role, use the
-<<privileges-to-setup-beats,setup role>> to pre-load dependencies. This section
-assumes that you've pre-loaded dependencies. 
+[[kibana-user-privileges]]
+=== Grant privileges and roles needed to read APM data in {kib}
 
-ifndef::no_ilm[]
-When using ILM, turn off the ILM setup check in the {beatname_uc} config file before
-running {beatname_uc} to publish events:
+++++
+<titleabbrev>Create a _reader_ user</titleabbrev>
+++++
 
-[source,yaml]
-----
-setup.ilm.check_exists: false
-----
-endif::no_ilm[]
+{kib} users typically need to view apps, dashboards, and visualizations that contain
+APM data. These users might also need to create and edit dashboards, visualizations, and machine learning jobs.
 
-To grant the required privileges:
+To grant users the required privileges:
 
-. Create a *writer role*, called something like +{beat_default_index_prefix}_writer+,
-that has the following privileges:
+. Assign the following built-in roles to users who need to read APM data:
++
+[options="header"]
+|====
+|Role | Purpose
+
+|`kibana_user`
+|Use the APM UI
```

Comment:
The `kibana_user` is deprecated (sorry forgot to mention that) and should be replaced by the `kibana_admin`. When granting the `kibana_admin` one already grants all the privileges for Kibana spaces. This makes for a nice getting-started experience. 
In the section below, where pointing out more fine granular privileges, please ensure to note that those can be used combined with the `apm_user` instead of granting the whole `kibana_user`/`kibana_admin` role.

Reply:
You did mention it, I just forgot 😨. Thanks for the reminder. Good point on the kibana_admin. I've clarified.